src/Resources/contao/classes/DataContainer.php

```diff
@@ -1027,4 +1064,38 @@ protected function getPickerInputField($value, $attributes='')
 	 * @throws \Exception
 	 */
 	abstract protected function save($varValue);
+
+
+	/**
+	 * Create a data container object in the picker environment
+	 *
+	 * @return DataContainer
+	 */
+	private function getDataContainer()
```

Comment:
Well, it returns a `DataContainer` object, so the name seems suitable to me. Also, it is a private method (unless we want to change that?).